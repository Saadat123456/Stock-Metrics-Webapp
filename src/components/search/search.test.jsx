import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import { BrowserRouter as Router } from 'react-router-dom';
import Search from './search';
import store from '../../redux/configStore';

test('Renders Correctly', () => {
  const { container } = render(
    <Provider store={store}>
      <Router>
        <Search />
      </Router>
    </Provider>,
  );
  expect(container.firstChild).toMatchSnapshot();
});
