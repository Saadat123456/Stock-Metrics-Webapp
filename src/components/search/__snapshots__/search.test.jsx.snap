// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Correctly 1`] = `
<input
  id="search"
  style="width: 100%; padding: 4px 8px; outline: none;"
  type="text"
/>
`;
